### PR TITLE
Move test data in their own folder/file

### PR DESCRIPTION
**Pull request by Geod24**
This makes the TS source code more readable by separating those giant strings.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -113,4 +113,4 @@ database
 src/modules/data/types
 
 # Database files
-data/
+/data/
```
Comment (TrustHenry) [inline .gitignore:116]: Isn't it a folder?

Comment (Geod24) [reply .gitignore:116]: It means everything under this path. `data/` match any data folder in the repo (including `tests/data/`) while `/data/` only match `data/` at the root of the repository.

Comment (TrustHenry) [reply .gitignore:116]: Yes,
But in my case the root data folder at root `/data/` was not ignored by git.

Comment (Geod24) [reply .gitignore:116]: Could you demonstrate how it doesn't work for you ?

Comment (TrustHenry) [reply .gitignore:116]: https://github.com/bpfkorea/stoa/pull/156
There used to be `stoa/logs/` in front of.
I think it was a different case.

File: tests/Utils.ts [hunk 2]
```diff
@@ -12,13 +12,15 @@
...
+export const sample_data_raw = (() => {
+    return [
+        fs.readFileSync('tests/data/Block.0.sample1.json', 'utf-8'),
```
Comment (TrustHenry) [inline tests/Utils.ts:20]: Nice!

**Review by TrustHenry — APPROVED** [review]

**Pull request merged**